app/app.js

```diff
@@ -3,6 +3,8 @@ const path = require('path');
 const bodyParser = require('body-parser');
 const { User, Entry } = require('./models/models.js')
 const debug = process.env.DEBUG || false;
+const jwt = require('jsonwebtoken')
+const helpers = require('./auth/auth.js')
```

Comment:
be explicit about what you're pulling in: { jwtAuth, pwdAuth, jwtOptions } = require....

that way you can use them on their own, and it also tells people what to look for in the import from file